jdi-light/src/main/java/com/epam/jdi/light/actions/ActionHelper.java

```diff
@@ -331,15 +397,64 @@ static String getDefaultName(String method, MapArray<String, Object> args) {
     static MapArray<String, Object> classFields(Object obj) {
         return obj != null ? getAllFields(obj) : new MapArray<>();
     }
+    static MapArray<String, Object> classMethods(Object obj) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5478709)